icubam/db/synchronizer.py

```diff
@@ -81,8 +82,12 @@ def sync_icus(self, icus_df, force_update=False):
 
       # If an ICU exists with the same name, update:
       if db_icu is not None:
-        manager = self._managers.get(db_icu.icu_id, self._default_admin)[0]
+        try:
+          manager = self._managers.get(db_icu.icu_id, self._default_admin)[0]
+        except TypeError:
+          manager = self._managers.get(db_icu.icu_id, self._default_admin)
```

Comment:
Si is the issue that self._managers.get returns None, and then  `None[0]` is a type error?

Maybe more explicitly then,
```py
manager = self._managers.get(db_icu.icu_id, self._default_admin)
if manager is not None:
   manager = manager[0]
```